example/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link href="ext/codemirror.css" rel="stylesheet" type="text/css">
```

Comment:
@sccolbert, you're the one who created the `ext/` folder, what are your thoughts?